influx_grafana.yml

```diff
@@ -0,0 +1,21 @@
+- hosts: "{{ hosts_prefix }}-influx-grafana*"
+  sudo: yes
+  vars:
+    influxdb_version: latest
+  tasks: # The influxdb and telegraf packages disagree on ownership of /var/run/influxdb - Telegraf Issue #37
+    - name: Create /etc/default/telegraf to fix conflict between influxdb and telegraf packages
+      copy: dest=/etc/default/telegraf mode=0644 content="USER=influxdb\nGROUP=influxdb\n"
+  roles:
+    - role: influxdb
+    - role: telegraf
+    - role: Stouts.grafana
+  post_tasks:
+    - name: Copy grafana API control script
+      copy: src=datasource.sh dest=/usr/local/bin/datasource.sh mode=0750 owner=root group=root
```

Comment:
I got this error:

```yaml
TASK: [Copy grafana API control script] *************************************** 
failed: [hector070715-influx-grafana] => {"failed": true}
msg: could not find src=/Users/keymon/gds/tsuru-ansible/datasource.sh

FATAL: all hosts have already failed -- aborting
```

But works if I change this line to add the path `files/`:

```diff
$ git diff
diff --git a/influx_grafana.yml b/influx_grafana.yml
index c4405dc..51a3d7d 100644
--- a/influx_grafana.yml
+++ b/influx_grafana.yml
@@ -11,7 +11,7 @@
     - role: Stouts.grafana
   post_tasks:
     - name: Copy grafana API control script
-      copy: src=datasource.sh dest=/usr/local/bin/datasource.sh mode=0750 owner=root group=root
+      copy: src=files/datasource.sh dest=/usr/local/bin/datasource.sh mode=0750 owner=root group=root
     - name: Add influx datasource to grafana
       shell: /usr/local/bin/datasource.sh {{ influxdb_database }}
 
```